bot/exts/help_channels/_cog.py

```diff
@@ -118,6 +122,9 @@ async def claim_channel(self, message: discord.Message) -> None:
 
         await _caches.unanswered.set(message.channel.id, True)
 
+        # Removing the help channel from the dynamic message, and editing/sending that message.
+        self.available_help_channels.remove(message.channel)
+
```

Comment:
I think this forgets to call the update function?

Reply:
Actually, the call is not needed here. Works perfectly fine without it.